src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -111,6 +113,35 @@ public LibraryServices(ProtoCore.Core libraryManagementCore, IPathManager pathMa
             PopulateOperators();
             PopulatePreloadLibraries();
             LibraryLoadFailed += new EventHandler<LibraryLoadFailedEventArgs>(LibraryLoadFailureHandler);
+
+            //AppDomain.CurrentDomain.AssemblyResolve += ResolveAssembly;
```

Comment:
@mjkkirschner this is the handler I was trying to register here but the event never reaches here as it encounters an exception in the other handler that is registered before this one: https://github.com/DynamoDS/Dynamo/blob/bd9ae7cb3c46ed61070daf823c81a69f6e7fa411/src/DynamoSandbox/Program.cs#L41